server/routerlicious/packages/routerlicious-base/src/riddler/api.ts

```diff
@@ -128,7 +146,8 @@ export function create(
 	router.put("/tenants/:id/key", (request, response) => {
 		const tenantId = request.params.id;
 		const keyName = request.body.keyName as string;
-		const refreshKeyP = manager.refreshTenantKey(tenantId, keyName);
+		const refreshPrivateKey = request.body.refreshPrivateKey as boolean;
```

Comment:
The tenant should not be able to send us a request to "refreshPrivateKey". 
"refreshPrivateKey" should be an internal method not exposed to the tenant.

Reply:
Yes, the `/tenants/:id/key` API with `refreshPrivateKey` as `true` would only be called by internal APIs that will refresh the private tenant keys periodically.